.github/workflows/platform-release.yaml

```
@@ -145,29 +146,61 @@ jobs:
             retention-days: 7
             path: |
               ./dist/*.tar.gz
-
-  # Creates a release with the artifacts from the previous steps.
+  
+  # Creates a release using artifacts created in the previous steps.
   # workflow_dispatch triggered versions will be draft releases.
   # CLI docs are not updated for workflow_dispatch triggered versions.
   release:
     runs-on: ubuntu-latest
     needs: [build-linux, build-macos, build-windows]
     steps:
-      - name: Merge built artifacts
+      # checkout the current repository
+      - name: Checkout kitops
+        uses: actions/checkout@eef61447b9ff4aafe5dcd4e0bbf5d482be7e7871 # v4.2.1
+        with:
+          path: kitops
+      
+      # Create the homebrew-file directory to store the 
+      # homebrew artifacts
+      - name: Create homebrew-files dir
+        run: mkdir homebrew-files
+
+     # Copy the awk script and template to the homebrew-files dir.
+     # These will be used later to build the homebrew recipe file.
+      - name: Copy homebrew artifacts to homebrew-files dir
+        run: cp ./kitops/build/homebrew/*.* ./homebrew-files
```

This can just be one step for simplicity
```suggestion
     # Copy the awk script and template to the homebrew-files dir.
     # These will be used later to build the homebrew recipe file.
      - name: Copy homebrew artifacts to homebrew-files dir
        run: |
          mkdir -p homebrew-files
          cp ./kitops/build/homebrew/*.* ./homebrew-files
```